content/exchange/artifacts/USBYara.yaml

```diff
@@ -0,0 +1,56 @@
+name: Windows.Detection.USBYara
+author: Matt Green - @mgreen27
+description: |
+  Run yara over usb when USB is plugged into machine.
+  Return context and hit details.
+
+  This artifact requires:  
+    Windows.Monitor.USBPlugIn  
+    Generic.Detection.Yara.Glob  
+
+  Yara rule deployed and target path can be modified.
+  
+type: CLIENT_EVENT
+
+parameters:
+  - name: TargetGlob
+    default: '/**.lnk'
+  - name: PayloadYara
+    default: |
+            rule recyclebin_lnk{
+                meta:
+                    description = "AvastSvcpCP lnkfile"
+                    author = "@mgreen27"
+                    date = "2021-11-18"
+            
+                strings:
+                    $s1 = "AvastSvcpCP" wide nocase
+                    $s2 = "cefhelper.exe" wide nocase
+                    $s3 = "RECYCLER.BIN" wide nocase
+                    $s4 = "wsc.zip" wide nocase
+                    $s6 = "/q /c" wide nocase 
+                    $s8 = "S-1-5-21-1063499884-3365855816-3691837489-1000" wide nocase 
+                    $s9 = "Xayemarlwin-pc" wide nocase 
+            
+               condition:
+                    int16(0) == 0x004c and any of them
+            }
+            
+sources:
+  - query: |
+      SELECT * FROM foreach(
+            row={ SELECT DriveName,TimeCreated FROM Artifact.Windows.Monitor.USBPlugIn() },
```

Comment:
So we do have problems importing other exchange artifacts because the import adds a prefix.

Reply:
Thats a good point. I have been importing using "Windows." as a prefix and didnt even think about that. Do you think we should add "Exchange." here or add the USBMonitor to the main repo?

Reply:
I think it is better to add Exchange here - but we do need to think about it - the import artifact allows us to change the prefix so maybe we also need to update internal references. 

In any case in the short term adding an exchange prefix here will make it all work out of the box :-)

Reply:
just updated :)